paf-mvp-demo-express/src/views/advertiser/index.hbs

```diff
@@ -1,10 +1,20 @@
 <!DOCTYPE html>
 <html lang="html">
 <head>
+
+{{#if cmp}}
+
+    <!-- Preload the JavaScript needed for the CMP which is used as soon as the body element starts to render -->
+    <link rel="preload" as="script" href="https://{{cdnHost}}/assets/cmp/ok-ui.min.js">
+
+{{else}}
+
     <!-- PAF integration -->
     <link rel="preload" href="https://{{cdnHost}}/assets/app.bundle.js" as="script" />
     <link rel="preload" href="https://{{cdnHost}}/assets/paf-lib.js" as="script" />
 
+{{/if}}
```

Comment:
There is going to be an issue with a link at the bottom of this page (and for the publisher page):
https://github.com/prebid/paf-mvp-implementation/pull/120/files?short_path=496599c#diff-d24fbc51956b593b0c1bc751069abf04ec73cd6349d63e84a800c41cb4c673f3L796
Is a link for website visitors to edit their existing preferences. Today it is linked to the PAF UI.
It needs to handle the CMP UI as well.

Reply:
Can the bottom link be changed to window.PAFUI.showPrompt ?

Reply:
The change to use showPrompt has been made in the next push.
There is a related question about how and how often the publisher local data is revalidated with OneKey. This doesn't relate to this PR however.